packages/shared/src/components/Feed.spec.tsx

```diff
@@ -423,7 +423,6 @@ describe('Feed logged in', () => {
         edges: [defaultFeedPage.edges[0]],
       }),
     ]);
-    await screen.findByTestId('adItem');
```

Comment:
With current logic even though `adSpot` would be 2 (which means only show ads after 2 posts) we would still show the ad after only once post.

Example with feed page with only one post:
![image](https://github.com/user-attachments/assets/4d094fd0-fd0e-481b-91d2-75de8f25b5ed)

With new logic it correctly does not show ad until enough posts are listed:
![image](https://github.com/user-attachments/assets/b83bf491-f7c1-4247-bac4-95beced35078)

Verified with @idoshamun its ok behaviour.